### ISSUE TITLE
Assets cannot be easily shared and found

### ISSUE THREAD

**Issue by wborn** [openremote/openremote#1431]
It's currently difficult to refer to specific assets in e-mails or issues.

E.g. if you copy the URL of an asset (https://demo.openremote.app/manager/#/assets/false/4w1CElB2Olsn17uoKA1ktn) and paste it in a new browser tab it doesn't open the asset details screen.

This makes it hard to refer to assets.

If you have the asset ID you currently either have to use API calls or check the database to figure out the asset name and then use that to search for the asset.

**Comment by richturner** [comment]
The problem here is that there is no realm in the URL, we would need some way of parsing the realm in the URL either using existing queryparam mechanism e.g. `?realm=master` or some mod rewrite rule, or something else.

Also there are some weird bugs with the router that detects window navigator changes as when you directly paste an asset ID into there you have to presss enter twice for it to actually show that asset, could be this issue is experiencing this other issue also.

**Comment by wborn** [comment]
Yes it only seems to be an issue if you are logged into the master realm and copy/paste URLs of assets in other realms.
